src/backend/kvm/hasher.rs

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use std::convert::TryFrom;
+
+use anyhow::{Error, Result};
+
+pub struct Hasher(());
+
+impl TryFrom<super::config::Config> for Hasher {
+    type Error = Error;
+
+    #[inline]
+    fn try_from(_config: super::config::Config) -> Result<Self> {
+        Ok(Self(()))
+    }
+}
+
+impl super::super::Mapper for Hasher {
+    type Config = super::config::Config;
+    type Output = Vec<u8>;
+
+    #[inline]
+    fn map(
+        &mut self,
+        _pages: mmarinus::Map<mmarinus::perms::ReadWrite>,
+        _to: usize,
+        _with: u32,
+    ) -> anyhow::Result<()> {
+        Ok(())
+    }
+}
+
+impl TryFrom<Hasher> for Vec<u8> {
+    type Error = Error;
+
+    #[inline]
+    fn try_from(_hasher: Hasher) -> Result<Self> {
+        Ok(vec![0, 0, 0, 0, 0, 0, 0, 0])
+    }
+}
```

Comment:
This whole file is probably unnecessary.

Reply:
true